### ISSUE TITLE
Rename `zetaclient` to `zetasigner`

### ISSUE THREAD

**Issue by fadeev** [zeta-chain/node#1895]
`zetaclient` is not very descriptive: when someone says "blockchain client", you think of a library to interact with the blockchain (from a web or mobile app), or a GUI for end-users.

**Comment by lumtis** [comment]
I think `zetaclient` is more suitable since it's more general, the observer also observes txs on top of signing.
Actually the most explicit name for the program would simply be `observer` to me.

> zetaclient is not very descriptive: when someone says "blockchain client", you think of a library to interact with the blockchain (from a web or mobile app), or a GUI for end-users.

I think a client can refers to any program that interacts with a blockchain independently of the purpose, the `zeta` prefix precises that it is a zetachain specific client.

**Comment by fadeev** [comment]
@lumtis I agree that technically it is a client, but there are around 15 total users of zetaclient, it's a very general-purpose name for a very niche program.

I like `zetaobserver`!
